front_end/panels/ai_chat/LLM/LLMProviderRegistry.ts

```
@@ -5,6 +5,14 @@
 import { createLogger } from '../core/Logger.js';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)